shell/platform/android/io/flutter/view/FlutterView.java

```diff
@@ -997,9 +997,16 @@ public void onReceive(Context context, Intent intent) {
             this.surfaceTexture.setOnFrameAvailableListener(new SurfaceTexture.OnFrameAvailableListener() {
                 @Override
                 public void onFrameAvailable(SurfaceTexture texture) {
+                    if (released) {
+                        // Even though we make sure to unregister the callback before releasing,
+                        // SurfaceTexture has a data race when accessing the callback, so the callback may
+                        // still be called by a stale reference after released==true and mNativeView==null.
+                        // See b/113099095.
```

Comment:
Can we avoid referencing buganizer from the framework?

Reply:
If necessary, please copy the buganizer bug over to GitHub and then reference that here instead.

Reply:
Removed the bug reference. (FWIW, the buganizer bug is against Android base system).